swap/src/bitcoin/cancel.rs

```diff
@@ -149,7 +149,39 @@ impl TxCancel {
         OutPoint::new(self.inner.txid(), 0)
     }
 
-    pub fn add_signatures(
+    pub fn complete_as_alice(
```

Comment:
I think we should add a SignedTransaction type here. Right now it is not clear to me what complete means

Reply:
Agreed that `complete` is not the best name. I chose it for consistency with the other types. Probably should rename it everywhere :)

Open to suggestions here. Could probably also use a doc comment to explain what this does.